### PR TITLE
Note deprecation of `<param>` element

### PR DESCRIPTION
**Pull request by DmitryBarskov**
### Description

Leave a note about deprecation of `<param>` element on [Void element page](https://developer.mozilla.org/en-US/docs/Glossary/Void_element)

### Motivation

Highlight the deprecation to make navigation easier.

### Additional details

Deprecation of `<param>` is stated [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/param) for instance.

### CODE REVIEW

File: files/en-us/glossary/void_element/index.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ The void elements in HTML are as follows:
 - {{HTMLElement("hr")}}
 - {{HTMLElement("img")}}
 - {{HTMLElement("input")}}
-- {{HTMLElement("keygen")}}(HTML 5.2 Draft removed)
+- {{HTMLElement("keygen")}} (HTML 5.2 Draft removed)
 - {{HTMLElement("link")}}
 - {{HTMLElement("meta")}}
-- {{HTMLElement("param")}}
+- {{HTMLElement("param")}} (Deprecated element)
```
Comment (Josh-Cena) [inline files/en-us/glossary/void_element/index.md:32]:
```suggestion
- {{HTMLElement("param")}} {{deprecated_inline}}
```

File: files/en-us/glossary/void_element/index.md [hunk 2]
```diff
@@ -26,10 +26,10 @@ The void elements in HTML are as follows:
 - {{HTMLElement("hr")}}
 - {{HTMLElement("img")}}
 - {{HTMLElement("input")}}
-- {{HTMLElement("keygen")}}(HTML 5.2 Draft removed)
+- {{HTMLElement("keygen")}} (HTML 5.2 Draft removed)
```
Comment (teoli2003) [inline files/en-us/glossary/void_element/index.md:29]: `<keygen>` is deprecated too.

Comment (teoli2003) [reply files/en-us/glossary/void_element/index.md:29]:
```suggestion
- {{HTMLElement("keygen")}} {{deprecated_inline}}
```

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! 🎉 

(If you are looking to contribute further, here is a list of possible tasks: https://github.com/orgs/mdn/projects/25 )

**Pull request merged**